cost/google/cud_recommendations/CHANGELOG.md

```diff
@@ -24,3 +24,7 @@
 - Change to template name from "Google Committed Use Discount Recommender Policy" to "Google Committed Use Discount Recommender"
 
 - updates to output of cud recommendations savings, addition of monthly calculations
+
+## v3.1
+
+- Fixed missing savings field
```

Comment:
Maybe put newest versions at the top?
E.g.: https://github.com/flexera-public/policy_templates/blob/master/cost/aws/unused_rds/CHANGELOG.md